sphinx/conf.py

```diff
@@ -0,0 +1,68 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+import sphinx.builders.html
+import sphinx.builders.latex
+import sphinx.builders.texinfo
+import sphinx.builders.text
+import sphinx.ext.autodoc
+
+# Append project source directory to path environment variable
+sys.path.insert(0, os.path.abspath("../src"))
+
+django_settings = "voctrainer.settings"
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Lunes CMS'
+copyright = '2021, Lunes'
+author = 'Lunes'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0'
```

Comment:
This version should typically match the version defined in [setup.py](https://github.com/digitalfabrik/lunes-cms/blob/37541580c67e03e7801b2e7fbff1295c8419e879/setup.py#L9), and ideally also the latest GitHub release tag, but it seems that your versioning isn't consistent at the moment anyway? :sweat_smile:

Reply:
Indeed, we haven't discussed this topic yet :) I'll create an issue for that